cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceModal.js

```diff
@@ -418,7 +417,7 @@ export default class SetPreferenceModal extends Component {
             </div>
           :
             <div className="text-xs-center">
-              No Inherited Preferences
+              {T.translate('features.FastAction.setPreferencesNoInheritedPrefs')}
```

Comment:
it's probably better to nest these under `SetPreferences`. Long names like this is not very readable